guides/content/developer/core/taxation.md

```diff
@@ -105,16 +105,12 @@ Finally, if the taxable address (either the shipping or billing, depending on th
 
 Many jurisdictions have what is commonly referred to as a Value Added Tax (VAT.) In these cases the tax is typically applied to the price. This means that prices for items are "inclusive of tax" and no additional tax needs to be applied during checkout.
 
-In the case of tax inclusive pricing the store owner should enter all prices inclusive of tax. This makes it easy for Spree to display the tax inclusive price since it won't have to be constantly calculated on the fly.
+In the case of tax inclusive pricing the store owner can enter all prices inclusive of tax if she has set her home country as the default zone. If there is no default zone set, any taxes that are included in the price will be added to the net prices on the fly depending on the current order's tax zone.
```

Comment:
Okay for 'their'. 

When the current order already has a zone that differs from default zone (empty cart, address entered): 
There is a `Spree::Price#price_including_vat_for(zone)` method, which does that calculation - both on [display](https://github.com/magiclabs/spree/blob/refactor-tax-on-master/core/app/helpers/spree/base_helper.rb#L19-L21) and on [creating](https://github.com/magiclabs/spree/blob/refactor-tax-on-master/core/app/models/spree/line_item.rb#L13) [line items](https://github.com/magiclabs/spree/blob/refactor-tax-on-master/core/app/models/spree/line_item.rb#L37-L49). 

The prices are also updated [after the address step](https://github.com/magiclabs/spree/blob/refactor-tax-on-master/core/app/models/spree/order/checkout.rb#L90).